### PR TITLE
[DCSFUS-37] DATABASE DESIGN FOR A TRANSPORTATION COMPANY

### PR DESCRIPTION
**Pull request by bayooba**
This is my submission for the second task under the Databases module.
Attached is a screenshot of part of the SQL query statements. Thank you.

trans_comp_database

### CODE REVIEW

File: submissions/adebayoObafemi/database/transportation.sql [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+
+alter table trips
+modify passenger_class enum('1', '2', '3', ' ') DEFAULT ' ' not null;
+
+-- create the table for unplanned events such as accident--
+create table accidents (
+	id int not null auto_increment,
+    event_date date,
+    passenger_id int not null,
+    status_of_passenger enum('SURVIVED', 'DID NOT SURVIVE', ' ') DEFAULT ' ' not null,
```
Comment (Mayowa2020) [inline submissions/adebayoObafemi/database/transportation.sql:47]: Change line 47 to:  survivals BOOLEAN NOT NULL,

**Review by Mayowa2020 — CHANGES_REQUESTED** [review]
Well done @bayooba! Kindly make the necessary corrections and revert when done.

File: submissions/adebayoObafemi/database/transportation.sql (hunk 1 above)
Comment (Mayowa2020) [reply submissions/adebayoObafemi/database/transportation.sql:55]: Hello @bayooba, replace line 55 entirely with:  survivals BOOLEAN NOT NULL,
OR survived BOOLEAN NOT NULL. So, remove the entire statement and replace with the stated.

Please note that Boolean values are nothing but 0 and 1 i.e. either true or false. if the condition satisfies, it is true else it is false

**Review by Mayowa2020 — CHANGES_REQUESTED** [review]
Kindly make the requested change and revert when done

File: submissions/adebayoObafemi/database/transportation.sql [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+    primary key (id),
+    foreign key (passenger_id) references passengers (id)
+);
+
+-- after review: modify accidents table --
+alter table accidents
+drop column event_date;
+
+alter table accidents
+change status_of_passenger survivals boolean not null;
```
Comment (Mayowa2020) [inline submissions/adebayoObafemi/database/transportation.sql:65]: Hello @bayooba, did u push your changes? I can't see the last change I requested for.

**Review by Mayowa2020 — COMMENTED** [review]
Hello @bayooba, I can't see the updates, line 65 is the same. Kindly make the requested change and revert.

File: submissions/adebayoObafemi/database/transportation.sql (hunk 2 above)
Comment (bayooba) [reply submissions/adebayoObafemi/database/transportation.sql:65]: Hello @Mayowa2020, I have made the change. You requested that I changed line 55 https://github.com/Dufuna-CodeCamp/Dufuna-CodeCamp-21/pull/2273/commits/f4913e88afcb4c00c1bf1aeff0c53de49699495e#r1024496807

Line 65 is the SQL query statement that was executed to make the changes. However I made the edited the file directly here on GitHub and didn't make a PR through git bash.

**Pull request closed without merging**